x/auth/ante_test.go

```diff
@@ -52,17 +52,23 @@ func checkInvalidTx(t *testing.T, anteHandler sdk.AnteHandler, ctx sdk.Context,
 	assert.Equal(t, sdk.ToABCICode(sdk.CodespaceRoot, code), result.Code)
 }
 
-func newTestTx(ctx sdk.Context, msg sdk.Msg, privs []crypto.PrivKey, accNums []int64, seqs []int64, fee StdFee) sdk.Tx {
-	signBytes := StdSignBytes(ctx.ChainID(), accNums, seqs, fee, msg)
-	return newTestTxWithSignBytes(msg, privs, accNums, seqs, fee, signBytes)
+func newTestTx(ctx sdk.Context, msgs []sdk.Msg, privs []crypto.PrivKey, accNums []int64, seqs []int64, fee StdFee) sdk.Tx {
```

Comment:
Can we add a new test transaction type with multiple required signers (with different account numbers / sequences) and test all the signature validity checking for that case?

Reply:
Don't believe we need a new test transaction type for that? The signers, account numbers, and sequences are accumulated from the msgs passed in.

Tests for this were strewn across the test file. In my latest commit, I have a seperate function that tests multimsg signer functionality with different account numbers/sequences.